airflow/providers/elasticsearch/CHANGELOG.rst

```diff
@@ -21,15 +21,18 @@
    and you want to add an explanation to the users on how they are supposed to deal with them.
    The changelog is updated and maintained semi-automatically by release manager.
 
-Changelog
----------
+Changelog for ``apache-airflow-providers-elasticsearch``
+--------------------------------------------------------
 
 5.0.0
 .....
 
 .. note::
   Deprecate non-official elasticsearch libraries. Only the official elasticsearch library was used
 
+.. Review and move the new changes to one of the sections above:
+   * ``Deprecate the 2 non-official elasticsearch libraries (#31920)``
```

Comment:
Was it intentional to gengerate the release notes for unreleased version?

Reply:
Nope. Let me review it again - I could have accidentally added it, so let me review all og them

Reply:
Removed and checked that only header changes are in the CHANGELOGS now.